platform/features/imagery/src/controllers/ImageryController.js

```diff
@@ -75,50 +92,133 @@ define(
                     this.unsubscribe = this.openmct.telemetry
-                        .subscribe(this.domainObject, this.updateValues);
-                    this.openmct.telemetry
-                        .request(this.domainObject, {
-                            strategy: 'latest',
-                            size: 1
-                        })
-                        .then(function (values) {
-                            this.updateValues(values[0]);
+                        .subscribe(this.domainObject, function (datum) {
+                            this.updateHistory(datum);
+                            this.updateValues(datum);
                         }.bind(this));
+                    this.requestLad(false);
+                    this.requestHistory(this.openmct.time.bounds());
+                }.bind(this));
+        };
+
+        ImageryController.prototype.requestHistory = function (bounds) {
+            this.requestCount++;
+            this.$scope.imageHistory = [];
+            var requestId = this.requestCount;
+            this.openmct.telemetry
+                .request(this.domainObject, bounds)
+                    .then(function (values) {
+                        if (this.requestCount > requestId) {
+                            return Promise.reject('Stale request');
+                        }
+                        values.forEach(function (datum) {
+                            this.updateHistory(datum);
+                        }.bind(this));
+                        this.requestLad(true);
+                    }.bind(this));
+        };
+
+        // Optional addToHistory argument allows for two use cases:
+        //     updating url and timestamp only for standard imagery view,
+        //     i.e to populate the view before history is requested OR
+        //     appending to the running imagery history
+        ImageryController.prototype.requestLad = function (addToHistory) {
+            this.openmct.telemetry
+                .request(this.domainObject, {
+                    strategy: 'latest',
+                    size: 1
+                })
+                .then(function (values) {
+                    this.updateValues(values[0]);
+                    if (addToHistory !== false) {
+                        this.updateHistory(values[0]);
+                    }
                 }.bind(this));
         };
 
         ImageryController.prototype.stopListening = function () {
+            this.openmct.time.off('bounds', this.onBoundsChange);
+            this.scrollable.off('scroll', this.onScroll);
             if (this.unsubscribe) {
                 this.unsubscribe();
                 delete this.unsubscribe;
             }
         };
 
+        // Query for new historical data on manual bound change
+        ImageryController.prototype.onBoundsChange = function (newBounds, tick) {
+            if (this.domainObject && !tick &&
+                !_.isEqual(this.lastBound, newBounds)) {
```

Comment:
Now that https://github.com/nasa/openmct/pull/1638 is merged you can remove the isEqual checking here.  You won't get excessive bounds now.